adapters/Pangle/PangleAdapter/Bidding/GADPangleRTBNativeRenderer.m

```diff
@@ -0,0 +1,199 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import "GADPangleRTBNativeRenderer.h"
+#import "GADMediationAdapterPangleConstants.h"
+#import <BUAdSDK/BUAdSDK.h>
+#import "GADMAdapterPangleUtils.h"
+#include <stdatomic.h>
+#import "GADPangleNetworkExtras.h"
+
+static NSString *const BUDNativeAdTranslateKey = @"bu_nativeAd";
+
+@interface GADPangleRTBNativeRenderer()<BUNativeAdDelegate> {
+    /// The completion handler to call when the ad loading succeeds or fails.
+    GADMediationNativeLoadCompletionHandler _loadCompletionHandler;
+    /// The Pangle native ad.
+    BUNativeAd *_nativeAd;
+    /// The Pangle related view
+    BUNativeAdRelatedView *_relatedView;
+    /// An ad event delegate to invoke when ad rendering events occur.
+    id<GADMediationNativeAdEventDelegate> _delegate;
+
+}
+
+@end
+
+@implementation GADPangleRTBNativeRenderer
+@synthesize images = _images,icon = _icon;
```

Comment:
Prefer having these as properties, is there a specific reason of having `@synthesize` here?

Reply:
sorry, I will answer later

Reply:
`_image` & `_icon` will set in `-nativeAdDidLoad:view:`